compiler/damlc/stable-packages/BUILD.bazel

```diff
@@ -30,12 +30,16 @@ genrule(
         "daml-prim/GHC-Prim.dalf",
         "daml-prim/GHC-Tuple.dalf",
         "daml-prim/DA-Types.dalf",
+        "daml-stdlib/DA-Internal-Template.dalf",
+        "daml-stdlib/DA-Internal-Any.dalf",
     ],
     cmd = """
       $(location :generate-stable-package) --module GHC.Types -o $(location daml-prim/GHC-Types.dalf)
       $(location :generate-stable-package) --module GHC.Prim -o $(location daml-prim/GHC-Prim.dalf)
       $(location :generate-stable-package) --module GHC.Tuple -o $(location daml-prim/GHC-Tuple.dalf)
       $(location :generate-stable-package) --module DA.Types -o $(location daml-prim/DA-Types.dalf)
+      $(location :generate-stable-package) --module DA.Internal.Any -o $(location daml-stdlib/DA-Internal-Any.dalf)
```

Comment:
Do theis need to be stable given that the `Any*` types are not serializable?

Reply:
Yes, since they leak into typeclass signatures.

Reply:
Very good point. I'm glad you spotted that. Kudos!